src/main/java/me/i509/fabric/bulkyshulkies/api/block/base/AbstractShulkerBoxBlock.java

```diff
@@ -279,7 +279,11 @@ public DyeColor getColor() {
 
 	protected abstract void openContainer(BlockPos pos, PlayerEntity playerEntity, Text displayName);
 
-	public interface SingleTypePropertyRetriever<T> {
+    public int getSlotCount(){
+    	return this.slotCount;
+	}
+
+    public interface SingleTypePropertyRetriever<T> {
```

Comment:
I use tabs for spacing in my code so you might want to change that.

Reply:
Will do

Reply:
This here still needs to be tabs, go set your IDE formatting to make a project code format for this and then make it defualt to tabs